src/views/Predictions/Positions.tsx

```diff
@@ -22,19 +23,23 @@ const StyledSwiper = styled.div`
     width: 320px;
   }
 `
-const Positions: React.FC = () => {
+const Positions: React.FC<{ view?: string }> = ({ view }) => {
```

Comment:
View needs to be `PageView`

Reply:
Yea also thought this - created a circular dependency importing that enum from `Mobile`. Where would you house non-state-related types? I can just create a types.ts at the view level?

Reply:
Lets create a `types.ts` file in `views/Predictions` folder. Not ideal I know...

Reply:
Sure thing. Pushed all these tweaks